### PR TITLE
Add 2018-12-15 Sao Paulo event

### PR DESCRIPTION
**Pull request by laisoliveira**
We'll have the second edition of ClojureBridge in São Paulo 🇧🇷  🎉

### CODE REVIEW

File: _posts/2018-12-15-sao-paulo.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+layout:             post
+permalink:          /events/2018-12-15-sao-paulo
+country:            Brazil
+city:               São Paulo
+street:             Capote Valente, 39
+latitude:           -23.5607203
+longitude:          -46.6734586
+start-date:         2018-12-14
+end-date:           2018-12-15
+registration-url:
```
Comment (laisoliveira) [inline _posts/2018-12-15-sao-paulo.md]: We're preparing a registration form, will update this 🔜

**Comment by vsmart** [comment]
Hi laisoliveira, @sassela and I will be your point of contact for this event! Please don't hesitate to reach out here or via email (info@clojurebridge.org) with any questions! 
Would you like us to publish this event update as-is, or wait until you have the registration info?

**Comment by laisoliveira** [comment]
Hello @vsmart and @sassela! Thank you for the support
I just updated with the registration url, it's ready to be published 🎉

**Comment by davidchambers** [comment]
:tada:

**Comment by davidchambers** [comment]
<https://clojurebridge.org/events/2018-12-15-sao-paulo>

**Comment by laisoliveira** [comment]
👩‍💻
Thank you @davidchambers @vsmart and @sassela  🎈 
 
I found this small typo in the event description, I hope it won't be too uch trouble to update it https://github.com/ClojureBridge/clojurebridge.github.io/pull/69

**Pull request merged**